gremlin-dotnet/src/Gremlin.Net/Process/Traversal/Strategy/Decoration/SubgraphStrategy.cs

```diff
@@ -53,6 +54,10 @@ public SubgraphStrategy() : base(JavaFqcn)
                 Configuration["edges"] = edges;
             if (vertexProperties != null)
                 Configuration["vertexProperties"] = vertexProperties;
+
+            // hate to write this by default so only going to write it if it's false which is the non-default value
```

Comment:
Are we 100% sure that `true` will stay the default? Otherwise, we should maybe really just always add this so we don't at some point change the default in Java to `false` and then forget to update this here.

Reply:
decided to change it to always write it.